components/popover/Popover.js

```diff
@@ -23,11 +23,11 @@ const factory = (axis, calculatePositions, Overlay) => {
       },
     };
 
-    setPlacementThrottled = () => throttle(this.setPlacement, 16);
-
     componentDidMount() {
       document.body.appendChild(this.popoverRoot);
 
+      this.setPlacementThrottled = throttle(this.setPlacement, 250);
```

Comment:
Why did it move to `componentDidMount` and why did you change it to 250ms? (16ms is to get approximately 60 fps)

Reply:
I put it in componentDidMount because when I put it outside, it threw an error, saying throttle expected a function. I just checked again, and if I put this line after the setPlacement() function, it does work. What would be the reasoning for not putting it in componentDidMount()?

I don't believe the repositioning calculation needs to happen at 60fps and I believe it might have an impact on performance if you have this happening on pages that also have a lot of things happening on resizing of the window.

Reply:
In my opinion it's a bit weird to put it in a lifecycle function, as it has nothing to do with the lifecycle. I would just put it after the `setPlacement` function (or in a constructor, but I prefer the first one).

Reply:
That makes sense, I moved the function.

 :ok: by @Kemosabert on 03/07/2018 14:18:32